.github/workflows/public_data.yml

```diff
@@ -15,6 +18,8 @@ jobs:
     - name: set env vars
       run: echo "::set-env name=CACHE_DATETIME::$(/bin/date -u "+%Y%m%d_%H%M%S")"
     - uses: actions/checkout@v2
+      with:
```

Comment:
Don't you want to checkout the current branch (which is perhaps `trigger-public-data`?)

Reply:
No, it needs to check out the master branch and run that code, because `trigger-public-data` will most likely fall behind in commits.

Reply:
Makes sense — can we add that as a comment to the yml file?